build/cmake/functions.cmake

```diff
@@ -294,16 +300,19 @@ function(wx_set_target_properties target_name is_base)
             )
     endif()
 
+    file(RELATIVE_PATH wxSETUP_HEADER_REL ${wxOUTPUT_DIR} ${wxSETUP_HEADER_PATH})
```

Comment:
What about debug build?

Reply:
`wxSETUP_HEADER_PATH` contains a generator expression for debug build. The result in the target file looks like:
```
INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/lib/vc_x64_lib/mswu\$<\$<CONFIG:Debug>:d>;${_IMPORT_PREFIX}/include"
```